src/tools/yarprobotstatepublisher/src/robotstatepublisher.cpp

```diff
@@ -19,7 +19,8 @@
 
 #include <yarp/math/Math.h>
 
-#include <iDynTree/Model/Model.h>
+// #include <iDynTree/Model/Model.h>
```

Comment:
```suggestion
```
If that is not used, I suggest to just remove it.

Reply:
Thanks, yes I forgot to remove it